training/hello_world_training.py

```diff
@@ -90,7 +101,6 @@ def make_default_playlist() -> Playlist:
         DrivesToBallExercise('Get close to ball'),
         DrivesToBallExercise('Get close-ish to ball', grader=DriveToBallGrader(min_dist_to_pass=1000))
     ]
-    for exercise in exercises:
-        exercise.match_config = make_match_config_with_my_bot()
+    add_my_bot_to_playlist(exercises)
```

Comment:
```suggestion
    return add_my_bot_to_playlist(exercises)
```
And delete next line